app/controllers/planning_applications/assessment/local_policy_areas_controller.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+module PlanningApplications
+  module Assessment
+    class LocalPolicyAreasController < AuthenticationController
+      before_action :set_planning_application
+      before_action :set_local_policy, only: %i[create destroy edit new show update]
```

Comment:
I think we can remove the `:only` option - that's all the actions isn't it?

Reply:
Can't believe that didn't occur to me, haha 🤦‍♀️